docs/PERMISSION_SETTING.md

```diff
@@ -0,0 +1,70 @@
+# Permission setting for non-root users' usage of Remoteproc Runtime
+
+For non-root users to use Remoteproc Runtime, the remoteproc driver and the container engine must be accessible for this user.
+
+## How to set Remoteproc Runtime to be accessible by non-root users
+
+### 1. Make remoteproc driver accessible to the user
+
+Usually, remoteproc driver can only be accessible to root. To change this setting, follow the below instructions:
+
+1. Create a group and add your user:
+
+   ```
```

Comment:
Code blocks should indicate the language where possible to improve syntax highlighting:
https://github.com/jincheng9/markdown_supported_languages?tab=readme-ov-file#heres-a-full-list-of-supported-languages
```suggestion
   ```sh
```